Packs/MicrosoftExchangeOnline/Integrations/EWSO365/EWSO365.py

```diff
@@ -2158,7 +2195,9 @@ def parse_incident_from_item(item):     # pragma: no cover
 
                             v = ' '.join(map(str.strip, v.split('\r\n')))
                             attached_email_headers.append((h.lower(), v))
+                        demisto.debug(f'{attached_email_headers=}')
                         for header in attachment.item.headers:
+                            demisto.debug(f'{header.name.lower()}, {header.value}')
```

Comment:
this could turn out too many IO operations for production, I suggest removing it (we print them all in 2198 anyway, right?)

Reply:
Yup, removed.